database_connection_setup.rb

```diff
@@ -0,0 +1,7 @@
+require './lib/database_connection'
+
+if ENV['ENVIRONMENT'] == 'test'
+  DatabaseConnection.setup('chitter_test')
+else
+  DatabaseConnection.setup('chitter')
+end
```

Comment:
Good use of test environment variables - if you decide to deploy the app, these need to be deleted otherwise Heroku complains.